infrastructure/README.md

```diff
@@ -0,0 +1,120 @@
+# infrastructure
+
+## Fly.io
+
+We are moved to [Fly.io](https://fly.io/) since https://github.com/helpwave/services/issues/164.
+
+### To-dos
+- Use FLY_APP_ID env for DAPR_APP_ID
```

Comment:
do issues exist for this?

Reply:
Not yet, a To-do to create these To-dos are on my To-do list. Lol.